src/react/App.js

```diff
@@ -56,20 +56,32 @@ const ButtonRow = props => {
   const [show, setShow] = useState(false);
```

Comment:
a page might have more pupups, so the naming should be more specific, like `showDeleteItemPopup` or something like that

Reply:
I dont get it.
what am i supposed to rename?
`<Popup
        show={show}
        handleClose={handleClose}
        header={header}
        body={body}
      ></Popup>
`
'Popup' is the general component name

Reply:
the variables in line 56, because each popup needs its own `const [show, setShow] ... ` basically :)